README.md

```diff
@@ -0,0 +1,41 @@
+# katello-pull-transport-migrate
+
+_katello-pull-transport-migrate_ is a script which can be run on a client to migrate from katello-agent to REX pull transport (formerly known as "pull provider.")
+
+## What it does
+* Gets configuration from `subscription-manager config --list`
+* Writes correct configuration in `config.toml` based on subscription-manager config
+* Enables pull transport daemon service (`yggdrasild`) on the client and displays status
+
+After a successful run, it is safe to remove katello-agent from the client.
+
+## Prerequisites
+
+### Server
+(the following soon to be done for you by foreman-installer - [#34239](https://projects.theforeman.org/issues/34239))
+
+* Message broker such as `mosquitto` running
+* smart_proxy_remote_execution_ssh configured as follows
+```yaml
+# /etc/foreman-proxy/settings.d/remote_execution_ssh.yml
+:mqtt_broker: localhost
+:mqtt_port: 1883
+:mode: pull-mqtt
+```
+
+### Client
+* `yggdrasil` and `foreman_ygg_worker` RPMs installed
+* Registered to Katello (script will exit if client is registered directly to RHSM)
+
+## Run
+
+```sh
+# set $SBINDIR if it's other than /usr/sbin
```

Comment:
I looked at this, but I don't really understand the need for this. Here is the only use I found:
https://github.com/theforeman/katello-pull-transport-migrate/blob/0a22ca27018f854e225a07b3aeb86ce7a42fba5f/katello-pull-transport-migrate#L11
Any process you call knows how it was called. In shell it's stored in `$0`. So if you change that line to ``then re-run `$0`.`` I'd expect it to work and you can get rid of `$SBINDIR` and be fully portable, which means it also works in development environments.

In fact, you can even add branding by creating a symlink to it (`ln -s satellite-pull-transport-migrate katello-pull-transport-migrate`). You may need to change this too:
https://github.com/theforeman/katello-pull-transport-migrate/blob/0a22ca27018f854e225a07b3aeb86ce7a42fba5f/katello-pull-transport-migrate#L49
There you may need to change it to `$(basename $0)`.

Reply:
While looking at the script I found several other things that should be improved. https://github.com/theforeman/katello-pull-transport-migrate/pull/2 implements those but doesn't implement the branding mentioned here.

Reply:
> you can even add branding

Per [this comment](https://github.com/theforeman/foreman-packaging/pull/7210#discussion_r771455930) from Eric we're not implementing branding yet, but I will keep it in mind.

Reply:
I think the `$SBINDIR` stuff was left over from [this discussion](https://github.com/theforeman/foreman-packaging/pull/7210#discussion_r765537913). You're right, it doesn't really do much. Removed and updated